WTWSMS/history/titles/c_sevilla.txt

```diff
@@ -9,14 +9,14 @@
 510.1.1={
 	holder=8600
 }
+551.1.1={
+    holder=36555
+	liege=0
+}
 554.1.1={
 	holder=36553 # Opimius Gaius Comenciolus 
 	liege = e_byzantium
 }
-572.1.1={ 
-	holder=36558 # Flavius Constans    
-	liege = 0
-}
 586.1.1={ 
```

Comment:
Why this change?

Reply:
It is historically accurate that king Athanagild rebelled in Seville that year. Also, it's impossible Flavius Constans held it for the same reasons as the others.